packages/twenty-front/src/modules/ui/input/components/Select.tsx

```diff
@@ -32,6 +39,7 @@ export type SelectProps<Value extends string | number | null> = {
   options: SelectOption<Value>[];
   value?: Value;
   withSearchInput?: boolean;
+  callToActionButton?: CallToActionButton;
```

Comment:
I'm not entirely convinced we should add specific use cases to the raw Select components. The Select component should allow composition: We should be able to add such a button quickly, but the Select component should not know about it.

One example I like is [Headless UI components](https://headlessui.com/react/combobox#basic-example):

![CleanShot 2024-10-08 at 15 51 36@2x](https://github.com/user-attachments/assets/1c346bc9-084b-4433-9c00-8b28c1e65ee2)

Another solution, if we want to add buttons in selects often, is to make it possible to inject buttons through the `options` property:

```tsx
options={[{ label: 'XX', value: "" }, { type: "divider" }, { type: "button", label: "Add one +", onClick: () => {}, icon: "" }]}
```

If we keep adding specific use cases like this to the Select component itself, it will quickly become bloated.

**I don't think we should change something to your PR, this is more a discussion to start with the whole team.**

Reply:
I Allowed that to myself as there already is a `SearchInput` option that can be displayed at the bottom of the select. @charlesBochet @thomtrp wdyt?